code/datums/supplypacks/hospitality_vr.dm

```diff
@@ -79,6 +79,3 @@
 	cost = 50
 	containertype = /obj/structure/closet/crate/freezer
 	containername = "Chinese takeout crate"
-
-/datum/supply_pack/randomised/hospitality/pizza
```

Comment:
The changes to pizza are being vetoed. This PR will not be accepted unless they are reverted.

Reply:
Statement redacted as Heroman made a good point.